### ISSUE TITLE
goreleaser init fails with no such file or directory

### ISSUE THREAD

**Issue by mentos1386** [goreleaser/goreleaser#1108]
**Describe the bug**

When trying to use `goreleaser init` it fails.

```
$ goreleaser init --debug       

   • Generating  file 
   ⨯ failed to init project    error=open : no such file or directory
```

**To Reproduce**

Run `goreleaser init` on an non-initialized repo. Without goreleaser config file.

**Expected behavior**

It should create goreleaser config file?

**Environment (please complete the following information):**

- OS: `linux manjaro`
- OS version: `Linux MANJARO #1 SMP PREEMPT Sun Jul 21 12:53:04 UTC 2019 x86_64 GNU/Linux`
- GoReleaser Version [goreleaser --version]
```
version: 0.114.0
commit: ab0e755a71afff0f13c5237f896ad86c8633a912
built at: 2019-08-06T11:52:31Z
built by: goreleaser
```
**Additional context**
I installed goreleaser from aur repository.

**Comment by caarlos0** [comment]
dupe of #1107

fix incoming, sorry about the delay

**Comment by caarlos0** [comment]
fixed in v0.114.1

**Comment by caarlos0** [comment]
thanks for bringing this up :)
